src/components/PreviousButton/PreviousButton.tsx

```diff
@@ -16,7 +15,17 @@ const Chevron = styled(ChevronLeft)`
   fill: ${themeColor('primary')};
 `
 
-const PreviousButton = ({ className, children, onClick }) => (
+export interface PreviousButtonProps {
```

Comment:
- [ ] no need export interface. export can be remove cause its only needed here.